File: java/program74.java
import java.io.*;

class program74
{
    public static int CountSpace(String str)
    {
        char Arr[] = str.toCharArray();
        int count = 0;
        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] == ' ')
            {
                count++;
            }
        }
        return count;
    }

    public static void main(String arg[])
    {
        BufferedReader bobj = new BufferedReader(new InputStreamReader(System.in));
        String str = "";
        try
        {
            System.out.println("Enter String");
            str = bobj.readLine();
        }
        catch(IOException eobj)
        {}
        finally
        {
            int ret = CountSpace(str);
            System.out.println("count of the white spaces are : "+ret);
        }
    }
}

File: cpp/program74.cpp
#include<iostream>
using namespace std;

int CountSpace(char *str)
{
    int iCount = 0;

    while(*str != '\0')
    {
        if(*str == ' ')
        {
            iCount++;
        }
        str++;
    }
    return iCount;
}

int main()
{
    char Arr[30];

    cout<<"Enter string \n";
    cin.getline(Arr,30);

    cout<<"Count of the space is : "<<CountSpace(Arr)<<"\n";

    return 0;
}